depends/packages/qt.mk

```diff
@@ -6,32 +6,29 @@ $(package)_file_name=$(package)-everywhere-opensource-src-$($(package)_version).
 $(package)_sha256_hash=84e924181d4ad6db00239d87250cc89868484a14841f77fb85ab1f1dbdcd7da1
 $(package)_dependencies=openssl
 $(package)_linux_dependencies=freetype fontconfig dbus libxcb libX11 xproto libXext
-$(package)_build_subdir=qtbase
-$(package)_qt_libs=corelib network widgets gui plugins testlib
+$(package)_qt_libs=corelib network gui
```

Comment:
While we're not going to use Qt5 just yet, I'd recommend going ahead and upgrading this to match BC Core PR [#6471](https://github.com/bitcoin/bitcoin/pull/6471), which upgrades Core to Qt 5.5.